kernel/sched/core.c

```diff
@@ -6,6 +6,7 @@
  *
  *  Copyright (C) 1991-2002  Linus Torvalds
  */
+#include "linux/rcupdate.h"
```

Comment:
Removed from patch 1 and added in patch 3. Otherwise the patchset LGTM.

Reply:
This line is not necessary so I removed it in the last commit.

Reply:
Why was is it added now? If its dead-code, then you can remove it from here as well and have a new commit with that!